spec/lib/gettext-setup/pot_spec.rb

```diff
@@ -144,4 +144,14 @@ def locales_path
       expect(new_contents).to eq(contents)
     end
   end
+  context 'Merge pot files' do
+    it 'Merges two pot files' do
+      completepot = GettextSetup.config['project_name'] + ".pot"
+      expect do
+        GettextSetup::Pot.merge_pot
```

Comment:
Alternatively, the `gettext_rake_spec.rb` test below might benefit even more from that test.